book/src/concepts/dispatcher.md

```diff
@@ -14,5 +14,4 @@ The dispatcher looks at all the `SystemData` in the systems and builds execution
 
 If you want to have the best performance possible, you should prefer immutable over mutable whenever it is possible. (`Read` instead of `Write`, `ReadStorage` instead of `WriteStorage`).
 
-_Note: Please however keep in mind that `Write` is still preferable to interior mutability, such as `Mutex` or `RwLock` for example.
-
+\_Note: Please however keep in mind that `Write` is still preferable to interior mutability, such as `Mutex` or `RwLock` for example.
```

Comment:
Unrelated to this PR, but this line doesn't make sense. `Write` _is_ interior mutability. Maybe this should be "preferable locks"? And then, there are cases where locks actually are preferable (we do use them for a few things).

Reply:
@torkleyy I agree, should we open an issue for that? Since this PR is unrelated?

Reply:
moved to https://github.com/amethyst/amethyst/issues/870